src/NuGetPackageVerifier/PackageIssueFactory.cs

```diff
@@ -209,5 +209,19 @@ public static PackageVerifierIssue AssemblyHasWrongJsonNetVersion(string assembl
                 string.Format("The assembly '{0}' references the wrong Json.NET version. Current version '{1}'; Expected version '{2}'.", assemblyPath, currentVersion, expectedVersion),
                 PackageIssueLevel.Error);
         }
+
+        public static PackageVerifierIssue IdDoesNotExist(string id)
+        {
+            return new PackageVerifierIssue(
+                "PACKAGE_OWNERSHIP",
+                id,
+                $"The id '{id}' does not exist on NuGet.org. See https://github.com/aspnet/Coherence-Signed/wiki/New-package-Id-creation for instructions.",
```

Comment:
The content at https://github.com/aspnet/Coherence-Signed/wiki/New-package-Id-creation looks good! Can you add info there about who to contact to upload the package? Very few people have permission, and I know I'm not one of them.